ci-operator/step-registry/aws/deprovision/route53/private-hosted-zone/aws-deprovision-route53-private-hosted-zone-commands.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+export AWS_SHARED_CREDENTIALS_FILE="${CLUSTER_PROFILE_DIR}/.awscred"
+REGION="${LEASED_RESOURCE}"
+HostedZoneId="${SHARED_DIR}/hosted_zone_id"
+
+echo "Deleting AWS route53 hosted zone"
+if [ ! -f "${HostedZoneId}" ]; then
+  echo "File ${HostedZoneId} does not exist."
+  exit 1
+fi
+
+cmd="aws --region $REGION route53 delete-hosted-zone --id $(cat ${HostedZoneId}) | jq -r '.ChangeInfo.Id' | awk -F '/' '{printf \$3}'"
+change_id=$(eval "${cmd}")
```

Comment:
why are you building a string to `eval`, instead of using:

```sh
HOSTED_ZONE="$(jq -r '.ChangeInfo.Id' "${HostedZoneId}" | awk -F '/' '{printf $3}')"
echo "Deleting AWS route53 hosted zone ${HOSTED_ZONE}"
CHANGE_ID="$(aws --region "${REGION}" route53 delete-hosted-zone --id "${HOSTED_ZONE})"
```

or similar?

Reply:
Done